client/src/ui/organisms/placement/index.tsx

```diff
@@ -92,7 +92,22 @@ export function SidePlacement() {
 function TopPlacementFallbackContent() {
   const gleanClick = useGleanClick();
 
-  return (
+  return Date.now() < Date.parse("2024-10-12") ? (
```

Comment:
Maybe inline the result?
```suggestion
  return Date.now() < 1728691200000 ? (
```

Reply:
I prefer the more readable version: I'd hope date parsing is pretty well optimised in JS engines